src/lib/sharing.ts

```diff
@@ -3,14 +3,17 @@ import {isIOS, isAndroid} from 'platform/detection'
 import Clipboard from '@react-native-clipboard/clipboard'
 import * as Toast from '../view/com/util/Toast'
 import {Share} from 'react-native'
+import {useLingui} from '@lingui/react'
+import {msg} from '@lingui/macro'
 
 /**
  * This function shares a URL using the native Share API if available, or copies it to the clipboard
  * and displays a toast message if not (mostly on web)
  * @param {string} url - A string representing the URL that needs to be shared or copied to the
  * clipboard.
  */
-export async function shareUrl(url: string) {
+export async function ShareUrl(url: string) {
```

Comment:
Why was this function capitalized? Usually that is reserved for React Components.

Reply:
It was reverted. After changing to the `t` macro, I no longer get the linting error, thanks.